docs/farming.md

```diff
@@ -226,19 +239,31 @@ Does not matter if the node/farmer executable is the previous one or from the ne
 The reason we require this is, with every snapshot change, the network might get partitioned, and you may be on a different genesis than the current one.
 In plain English, these commands are like a `reset` button for snapshot changes.
 
-## Glossary for useful farmer commands
-
-Structure -> `subspace-farmer-x86_64-*-snapshot <COMMAND>`
-
-- `farm --reward-address WALLET_ADDRESS` : starts background plotting and farming together, farmed testnet coins will be sent to `WALLET_ADDRESS`
-- `farm` : starts background plotting and farming together, rewards are sent to auto-generated wallet (see `identity` commands below)
-- `wipe` : erases the plot and identity (including plot, commitment, object mappings and identity files)
-- `identity import-from-mnemonic "spell out your seed phrase here"` : imports your existing identity from your seed phrase (not recommended! use `--reward-address` instead)
-- `identity view` : displays SS58 address (this is the same as `identity view --address`) where farmed testnet coins will be sent
-- `identity view --mnemonic` : displays mnemonic phrase of auto-generated wallet (sensitive information, keep this private, not very useful if `--reward-address` was used)
+## Farmer Commands
+
+Structure -> `./FARMER_FILE_NAME <COMMAND> <SUBCOMMAND>`
+
+- `erase-plot`: Erase existing plot (doesn't touch identity)
+  - `--custom-path <CUSTOM_PATH>`: Use custom path for data storage instead of platform-specific default.
+- `farm`: Start a farmer using previously created plot
+  - `--bootstrap-node <BOOTSTRAP_NODE>`: Multiaddrs of bootstrap node to connect to on startup, multiple are supported
```

Comment:
Again, I see you're trying to surface every parameter possible, but it is not helpful if people don't know what to specify there. Do you know which bootstrap nodes you need to specify in which case for instance? You could have removed all of this documentation and suggested to use `--help` instead with the same (and even better because of no maintenance cost) result.